packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -595,14 +580,12 @@ export class MgtFileList extends MgtTemplatedComponent {
           class="file-list"
           tabindex="0"
```

Comment:
I think that we need to remove the top level tab index here. these event handlers will get the events bubbled up from the list items